libs/application/templates/inheritance-report/src/fields/HeirsRepeater/index.tsx

```diff
@@ -62,7 +62,7 @@ export const HeirsRepeater: FC<
 
   const hasEstateMemberUnder18 = values.estate?.estateMembers?.some(
     (member: EstateMember) => {
-      const hasForeignCitizenship = member?.foreignCitizenship?.[0] === 'yes'
+      const hasForeignCitizenship = member?.foreignCitizenship?.[0] === YES
```

Comment:
seeing how this is being redeclared everywhere - put into a shared utility function if that does not already exist in the repo